packages/graphback-datasync/src/DataSyncPlugin.ts

```diff
@@ -22,10 +15,24 @@ export const DATASYNC_PLUGIN_NAME = "DataSyncPlugin";
  * It will generate delta queries
  */
 export class DataSyncPlugin extends GraphbackPlugin {
+  protected addVersion: boolean;
+
+  public constructor(addVersion: boolean = false) {
+    super()
+    this.addVersion = addVersion
+  }
 
   public transformSchema(metadata: GraphbackCoreMetadata): GraphQLSchema {
     const schema = metadata.getSchema()
     const schemaComposer = new SchemaComposer(schema);
+    if (!schemaComposer.has(GraphbackTimestamp.name)) {
+      schemaComposer.createScalarTC(GraphbackTimestamp);
+    } else {
+      const TSScalarTC = schemaComposer.getSTC(GraphbackTimestamp.name);
+      if (!isSpecifiedGraphbackScalarType(TSScalarTC.getType())) {
+        throw new Error('GraphbackTimestamp is not a Graphback scalar: DataSync requires GraphbackTimestamp to be a Graphback specified scalar.');
+      }
```

Comment:
I do not think this is necessary:

IIRC, under the hood `isSpecifiedGraphbackScalarType` checks if `GraphbackTimestamp` is among known Graphback scalars and if it is a scalar.

Reply:
Removed 🗑️